packages/core/package.json

```diff
@@ -32,6 +31,7 @@
     "file-loader": "^1.1.11",
     "key-mirror": "^1.0.1",
     "lodash": "^4.17.12",
+    "mapbox-gl": "^1.5.0",
```

Comment:
Can we add this to the list of `external` libraries in `rollup.config.js`?

Reply:
👌 added